subsys/usb/usb_descriptor.c

```diff
@@ -205,7 +209,11 @@ static struct dev_common_descriptor common_desc = {
 		.bConfigurationValue = 1,
 		.iConfiguration = 0,
 		.bmAttributes = USB_CONFIGURATION_ATTRIBUTES,
+#if (CONFIG_USB_SPEC_VERSION > 1)
```

Comment:
OK. I'll try to get some help from @finikorg as I am not too familiar with device tree.

Reply:
@jfischer-phytec-iot 
I think this is the only comment that is left to address. Is that correct?

Reply:
If we have default `bcdUSB` 2.0 do we still need this ? About DT, IMO we could use appropriate Kconfig.board and Kconfig.defconfig instead.

Reply:
I think yes, if the default device descriptor is used then it should have matching bMaxPower for the board, either from DT or Kconfig files.

Reply:
I too think that this is needed because high speed devices are allowed to draw a max of 500mA from the bus and the descriptor has to indicate that for the host to know.
Since I am not yet familiar with Device Tree, I added the config in Kconfig. Looks like @jfischer-phytec-iot and @finikorg agree Kconfig is fine. Thanks!